pyproject.toml

```diff
@@ -10,6 +10,7 @@ profile = "black"
 filterwarnings = [
     "error::DeprecationWarning",
     "error::FutureWarning",
+    "ignore::sklearn.exceptions.ConvergenceWarning",
```

Comment:
I wouldn't add this here, it's quite a bad one to completely ignore I'd say.

Reply:
I reverted it.